test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/ZookeeperK8sService.java

```diff
@@ -73,34 +73,34 @@ public void clean() {
     public boolean isRunning() {
 
         return k8sClient.getStatusOfPodWithLabel(NAMESPACE, "app", getID())
-                        .thenApply(statuses -> statuses.stream()
-                                                      .filter(podStatus -> podStatus.getContainerStatuses()
-                                                                                    .stream()
-                                                                                    .allMatch(st -> st.getState().getRunning() != null))
-                                                      .count())
-                        .thenApply(runCount -> runCount == DEFAULT_INSTANCE_COUNT)
-                        .exceptionally(t -> {
-                           log.warn("Exception observed while checking status of pod: {}. Details: {} ", getID(), t.getMessage());
-                           return false;
-                       }).join();
+                .thenApply(statuses -> statuses.stream()
+                        .filter(podStatus -> podStatus.getContainerStatuses()
+                                .stream()
+                                .allMatch(st -> st.getState().getRunning() != null))
+                        .count() * 2 - statuses.size())
+                .thenApply(netCount -> netCount > 0)
```

Comment:
Previously, it was expected that an exact number of instances are available, which places heavy restrictions on the compatible clusters that can be used (if not deployed via system test scripts). Change it so that only a majority of nodes are required to be in a running state.